packages/ember-codemod-remove-ember-css-modules/bin/ember-codemod-remove-ember-css-modules.ts

```diff
@@ -1,33 +1,34 @@
 #!/usr/bin/env node
+// eslint-disable-next-line n/shebang
 'use strict';
 
 import yargs from 'yargs';
 import { hideBin } from 'yargs/helpers';
 
 import { removeEmberCssModules } from '../src/index.js';
+import type { CodemodOptions } from '../src/types/index.js';
 
 // Provide a title to the process in `ps`
 process.title = 'ember-codemod-remove-ember-css-modules';
 
 // Set codemod options
-const { argv } = yargs(hideBin(process.argv))
+const argv = yargs(hideBin(process.argv))
   .option('component-structure', {
-    choices: ['flat', 'nested'],
+    choices: ['flat', 'nested'] as const,
     default: 'flat',
     describe: 'Component structure (how your components are colocated)',
     type: 'string',
   })
   .option('root', {
     describe: 'Location of your Ember project',
     type: 'string',
-  });
-
-function castEmptyStringToUndefined(string) {
-  return string === '' ? undefined : string;
-}
+  })
+  .parseSync();
 
-const codemodOptions = {
-  componentStructure: castEmptyStringToUndefined(argv['component-structure']),
+const codemodOptions: CodemodOptions = {
+  componentStructure: argv[
+    'component-structure'
+  ] as CodemodOptions['componentStructure'],
```

Comment:
Because of `default: 'flat'` in line 18, I don't think it's possible for `argv['component-structure']` to be the empty string `''` or `undefined`.

In other words, we assume that the value is `'flat'` or `'nested'`.